ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__arm64-nightly-4.17-upgrade-from-stable-4.11.yaml

```diff
@@ -83,6 +124,9 @@ tests:
   cron: 35 11 5 * *
   steps:
     cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:arm64-target
```

Comment:
Chain upgrade does not need `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE` setting.

Reply:
Oh, my fault, you are right, `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE` is needed.

Reply:
chain upgrade will drop last hop and use `cucushift-upgrade-toimage` for last hop upgrade, so it still need OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE, right?

https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh#L436